### PR TITLE
Use consular from PPA

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

**Comment by jerith** [comment]
This is an experimental PPA that I set up as a proof of concept. We should probably switch to one that isn't attached to my personal launchpad account before landing this change.

File: puppet/modules/consular/manifests/init.pp [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-  ~>
-  python::virtualenv { '/var/consular/venv':
-    requirements => '/var/consular/requirements.txt',
-  }
-  ~>
   file { '/etc/init/consular.conf':
     content => template('consular/init.consular.conf.erb'),
   }
   ~>
+  package { 'python-consular':
```
Comment (JayH5) [inline puppet/modules/consular/manifests/init.pp:13]: [Here](https://launchpad.net/~jerith/+archive/ubuntu/consular) it looks like the package is just called `consular`?

Comment (jerith) [inline puppet/modules/consular/manifests/init.pp:13]: `consular` is the name of the source package. The binary package is called `python-consular`.

**Comment by JayH5** [comment]
:+1:

**Comment by JayH5** [comment]
:+1: :smile:

**Pull request merged**